templates/misc/moonflower_ML_cupy_fft_smoothing.py

```diff
@@ -48,7 +48,7 @@
 # attach a reconstrucion engine
 p.engines = u.Param()
 p.engines.engine00 = u.Param()
-p.engines.engine00.name = 'ML_cupy'
+p.engines.engine00.name = 'ML_serial'
```

Comment:
@daurer did you mean to modify the cupy template to serial here?

Reply:
No, that was just for debugging. Should be reversed.